docs/getting-started/local-development.mdx

```diff
@@ -96,7 +96,7 @@ Copy the project slug from your project's [Connect page](https://console.ory.sh/
 Tunnel. Here we assume that your app runs on `http://localhost:3000`:
 
 ```shell
-ory tunnel --dev --project {project.id-or-project.slug} \
+ory tunnel --dev --project {project.slug} \
```

Comment:
```suggestion
ory tunnel --dev --project $PROJECT_SLUG \
```

lets use the same format as introduced in [this PR](https://github.com/ory/docs/pull/1727) just now so you can easily copy&paste with exported env vars.